### ISSUE TITLE
CNN classification，cost does not converge

### ISSUE THREAD

**Issue by yyydoing** [aqibsaeed/Urban-Sound-Classification#2]
With CNN classification, I run your code, but the loss function does not converge, what is the reason？Your map can be put to look at it

**Comment by aqibsaeed** [comment]
There can be several reasons for that. Following are the suggestions:  training large model, tuning learning rate, use momentum. Keep in mind this is not an easy to solve problem. Please look at the discussion on blog post related to that for more pointers.

**Comment by yyydoing** [comment]
When you run this “Urban Sound Classification using RNN” code yourself, is the loss function converge? i I would like to know the network structure is ok or that the extraction of the data feature is ok.

I modified the learning rate, adjusted the network structure, but still not convergence, I would like to know the results of your operation is convergence? The problem is that there is a problem with my operation or the code I am modifying

**Comment by aqibsaeed** [comment]
Very slowly! It's not that simple to let these model learn and converge. This problem is particularly hard in itself.
